app/controllers/addresses_controller.rb

```diff
@@ -0,0 +1,8 @@
+class AddressesController < APIController
+  before_action :doorkeeper_authorize!, only: [:index]
+
+  def index
+    authorize Address
+    render json: Address.all
```

Comment:
Should make a note to change this. Obviously will not perform.

Also, what's the order of operations on getting pins on a map? I don't believe it will be going through this controller, but by generating a walking list. I'm not sure this controller would ever be used directly in the current version.